lib/modules/@apostrophecms/ui/src/apos/scss/_zindex.scss

```diff
@@ -1,3 +1,4 @@
+$z-index-under: -1;
 $z-index-base: 0;
 $z-index-default: 1;
```

Comment:
Extremely minor but the default value for `z-index` corresponds to `0`. Maybe base should be called default and default should be something else?